python/graphscope/interactive/query.py

```diff
@@ -69,9 +85,29 @@ def graph_url(self):
         """The gremlin graph url can be used with any standard gremlin console, e.g., thinkerpop."""
         return self._graph_url
 
+    @property
+    def status(self):
+        return self._status
+
+    @status.setter
+    def status(self, value):
+        self._status = value
+
+    @property
+    def error_msg(self):
+        return self._error_msg
+
+    @error_msg.setter
+    def error_msg(self, error_msg):
+        self._error_msg = error_msg
+
+    def set_frontend(self, front_ip, front_port):
+        self._graph_url = "ws://%s:%d/gremlin" % (front_ip, front_port)
+        self._client = Client(self._graph_url, "g")
+
     def closed(self):
         """Return if the current instance is closed."""
-        return self._closed
+        return self._status == InteractiveQueryStatus.Closed
```

Comment:
Since you have three status type, what will happen if the status is `initializing` and `execute` is called?
Will the call be blocked? or error out?
ref: https://github.com/alibaba/GraphScope/blob/10f36b1dee462bf3efe9439a354e4f9e6d9f6da1/python/graphscope/interactive/query.py#L139-L141

Reply:
`excute` `subgraph` and `traversal_source` can only run under `running` status. Otherwise, raise an exception.